draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -4274,10 +4276,14 @@ Steps:
 38. return (xEn, xEd, yEn, yEd)
 ~~~
 
-### q = 3 (mod 4) {#ell2-map-to-3mod4}
+### Montgomery curves with q = 3 (mod 4) {#ell2-map-to-3mod4}
 
 The following is a straight-line implementation of Elligator 2
-that applies to any curve over GF(q) where q = 3 (mod 4).
+that applies to any Montgomery curve
+
+    K * t^2 = s^3 + J * s^2 + s
```

Comment:
I think we should revert this (and the change below) as it's redundant with the definition in 6.7.

Reply:
reverted